radicle-httpd/src/api/v1/projects.rs

```diff
@@ -1021,6 +1042,203 @@ mod routes {
         );
     }
 
+    #[tokio::test]
+    async fn test_projects_issues_create() {
+        let tmp = tempfile::tempdir().unwrap();
+        let ctx = test::seed(tmp.path());
+        let app = super::router(ctx.to_owned());
+        create_session(ctx).await;
+
+        let body = serde_json::to_vec(&json!({
+            "title": "Issue #2",
+            "description": "Change 'hello world' to 'hello everyone'",
+            "tags": ["bug"],
+        }))
+        .unwrap();
+        let response = post(
+            &app,
+            "/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/issues",
+            Some(Body::from(body)),
+            Some("u9MGAkkfkMOv0uDDB2WeUHBT7HbsO2Dy".to_string()),
+        )
+        .await;
+        assert_eq!(response.status(), StatusCode::CREATED);
+        assert_eq!(response.json().await, json!({ "success": true }));
```

Comment:
can you add a space between setups/actions and asserts?
```
…
.await;

assert_eq!(response.status(), StatusCode::CREATED);
…
```

Reply:
66674e4